### DISCUSSION TITLE
Fritz Repeater 6000

### DISCUSSION THREAD

**Discussion by thiscantbeserious** [Freetz-NG/freetz-ng#293] [Allgemein]
Ich hab mal den Source-Code von AVM angefordert. 

Laut der Liste ist der Repeater ja evtl. bereits lauffähig, oder? Was bedeutet der tag "[inhouse]"? 

Bin relativ experimentierfreudig, insofern werd ich nachher gleich mal einen Build anschmeissen, gibts irgendwas was ich im speziellen beachten sollte (ist mein erstes Gerät seit Langem von AVM)?

Für den Chipsatz hab ich für OpenWRT schon einen Kernel gefunden ...

Hab den Repeater schon hier vor mir stehen, aber mit der Stock-Firmware bleibt er echt hinter seiner (erwarteten) Leistung meilenweit zurück.

OT:

Die Ports sind übrigens wieder Unshielded (auch der 2.5 GbE Port) und für die Größe wird er relativ warm. Das 5.2GHZ Wifi scheint noch solala zu funktionieren - Reichweite ist nicht besonders hoch. Generell scheint die Signalqualität bei allen Geräten eher im Mittelfeld zu liegen, allerdings kommt er in der Praxis deutlich besser mit mehreren 4k/Livestreams zurecht als z. B. mein UniFi FlexHD. Sprich das Signal scheint zuverlässiger zu sein.

Bin noch nicht ganz sicher was ich von dem Teil halten soll ...

**Comment by fda77** [comment]
"inhaus" ist eine Erfindung von AVM. Da diese Firmwareimages nicht öffentlich sind (sic), wird auch keine GPL Source Code der kostenfrei deafür verwendet wurde veröffentlicht...
Eine normale "release" Vesion gibt es bislang nicht als Datei zum download, nur in den ausgelieferten Geräten.
Welche Firmware aktueller ist kann man an der "rev" (Revision) Angabe abschätzen

Ein Repeater mit 2,5Gbit Lan am durchschnittlichen deutschen max-bis-zu-30-mbit-telekom Breitbandanschluss? ^^
Für guten Empfang braucht man externe, ausrichtbare Antennen. Von AVM gibt es sowas nicht mehr

**Reply by thiscantbeserious** [reply]
🙉 Ja, den hätten sie eher als Power-AP verkaufen sollen, denn als Repeater. 

Inkl. entsprechender Firmware (Multi-SSID, VLAN-Tagging). 

Aber gut ...

Ist das Image dann eine vollständige Firmware - Freetz-NG - oder beschnittene Repeater-Firmware?

Also im speziellen das hier:

253.07.24 rev87819 {ALL} [PSQ19P2NL] (Inhaus)

Von hier:

https://freetz-ng.github.io/freetz-ng/FIRMWARES.html#newer-firmware-versions

So ganz steig ich noch nicht durch. Wie komm ich da ran? Bzw. wie funktioniert das?

Die Antwort von AVM kam auf jeden Fall schon. 

Quellcode wird in "bis zu 2 Wochen" bereitgestellt. Mal sehen ...

> Markus W (AVM)12.05.2021, 13:43 MESZ
> 
> Guten Tag Herr X,
> 
> vielen Dank für Ihr Interesse am Quellcode des FRITZ!Repeater 6000. Wir haben Ihre Anfrage weitergeleitet und melden uns, sobald die Datei zum Download bereitsteht. Dies kann bis zu zwei Wochen in Anspruch nehmen. 
> 
> Daher bitten wir Sie um ein wenig Geduld. | Markus W (AVM)

Den 2.5GbE Port hätten sie sich echt sparen können, wobei der vielleicht für Netzwerk-Interne Drahtlos-Backups interesannt ist - sofern man denn einen 2.5/10GbE-Switch und NAS hat. 

Gibt aber auch Kabel Schland 1 Pseudo-GbE Anschluss ... und in den Großstädten den echten Glasfaseranschluss. 

An unserer Adresse gibt's maximal 250 Mbits (Telekom, also vermutlich Super-Vectoring Schrott). 

Es wird aber besser (theoretisch).

Meinetwegen hätten sie den Port auch weglassen können und lieber vernünftige Features wie VLAN-Tagging von Haus aus freischalten können oder zumindest das Gäste WLAN fest getaggt per VLAN  ... ist mir etwas unverständlich bei der Hardware.

Aber das ist echt Jammern auf hohem Niveau, wenn ich mir anschaue was Ubiquiti mit ihrer Hardware verbricht, im spezifischen komme ich von einem MediaTek powered FlexHD der sehr viele (Power-)Funktionen hat - und bei stabilien Firmwarereleases dann 20-40% Paketverlusst einfährt und auch generell bei Last in die Knie geht (jetzt ist mir auch klar warum die nur 1 Jahr Garantie bieten und alles daran setzen die Hardware-Specs möglichst nirgends zu erwähnen). 

Und das wird nicht hotgefixt, sondern Wochen (oder waren es Monate lang?) ignoriert. 

Die Grundfunktion tritt da irgendwie in den Hintergrund, und das ist sprichwörtlich zum k*tzen wenn man drauf angewiesen ist. Dann lieber andersrum ...

Deswegen fällt das bei mir auch raus weil deren Wifi 6 APs ebenfalls auf MediaTek setzen ... bisher läuft der Repeater deutlich stabiler. Wenn auch beschnitten. 

Gibt halt absolut keine interesannte Wifi 6 Alternative (AC Wave 2 fällt raus) für mich die nur halbwegs bezahlbar wären und auch 3 Radios haben und entsprechend Leistungsreserve für den internen Datenaustausch bieten (die Betonung liegt auf - "möchte").

Ruckus, Cisco, ... da herrschen grade Preise das einem die Ohren schlackern. 600-1000€ für einen einzigen AP mit dem gleichen/ähnlichen Chipsatz.

Wohlgemerkt sind hier 12 Antennen verbaut - wenn auch nur Intern. Also 4 für jeden Chip.

Das Problem bei externen Antennen sind nicht nur die Verbindungsstücke sondern die riesigen Qualitättschwankungen - aus meiner persönlichen Erfahrung gesprochen. 

Das lässt sich bei interenen Antennen deutlich leichter kontrollieren - und bei der richtigen Bauform ergibt sich Strahlungsmustertechnisch sogar ein Vorteil da der Körper des APs in das Antennen-Design einbezogen werden kann.

Würde persönlich keinen AP mit externen Antennen mehr kaufen. 

Wenn dann gleich einen Outdoor-AP mit Richtantenne fest verbaut oder vorbereitet ... aber das brauch ich nicht weil ich nicht in Texas wohne und kein Grundstück mit mehreren Quadratkilometern Größe habe.

Eigentlich kauf ich normalerweise auch gar nichts von AVM weil ich nicht die Zeilgruppe bin - dass ist das erste Gerät überhaupt - auch nur wegen der Hardware.

Wobei ich sagen muss dass ich eine alte Fritzbox als DSL-Modem wegen dem guten DSL-Handling erwäge, im Vergleich zu meinen Erfahrungen mit TP-Link und Speedport, gerade in diesen Zeiten mit Netzübelastung mit höheren Packetverlussten (hatte eine Demo-Box da vom ISP).

**Reply by fda77** [reply]
Die aktuellste für den 6000 ist wohl `253.07.26 rev88564 {ALL} [PSQ19P2NL2-aM7Poh3a] (12.05.2021 10:41:40)`, die gibts über JUIS. Schaut so aus als ob die auch den Frag-Fritz Fix hat. Über JUIS gibts im IPPF sehr viel zu lesen, und mindestens 3 Arten es azufragen: Excel-Tabelle, Linux-Script und ein Windows-Programm. Eine INHAUS ist eine ganz normal Firmware, halt keine RELEASE sondere eher eine ALPHA die uU täglich aktualisiert wird. Der Source von der RELEASE passt evtl nicht zur INHAUS (zB Änderungen am Kernel), und die RELEASE gibt es nicht zum Download.
Da du scheinbar noch nie Freetz verwendest hat bin ich nciht sicher ob es eine gute Idee ist wenn du gleich mit einem Gerät anfängst das noch nie getestet wurde. Falls irgendwas nicht funktioniert müsstest du nämlich selbst schauen wo es hakt!

Auf das was die Telekom die andrehen will kann man absolut gar nichts geben! Mir wurde 100mbit angeboten, durch Drücker an der Haustür, telefonisch und auch schiftlich nachdem hier gefördert(!) "100" Mbit ausgebaut wurden. So einen Mistvertrag hatte ich 1 Woche, obwohl noch nich mal Fullsync von 50 Mbit erreicht wurde. Laut Hotline würde irgend welche rosane Magie von irgendwelchen sogenannten Experten ausgübt damit 100 möglich wären. War aber nix. Da erst "ausgebaut" wurde bleibt das die nächsten Jahrzehnte halt so... beim nächsten Umzug leg ich Wert auf Kabel-TV, das lief immer ohne Probleme

Mit AVM hast du IMMER Paketeverluste! Die benutzen nämlich nicht das ganz normale Routing von Linux sondern haben Dinger wie "multid" und "dsld" zusammengeschustert. Die machen Routing, DNS, DHCP etc. Ob die gut/sicher sind kann niemand sagen, da es keinen Quellcode gibt. Ganz sicher ist aber dass die Teile immer wieder seltsame Probleme verursachen. Versuch zB mal aus dem Internet von 2 verschiedenen IPs gleichzeitig die externe IP eine Fritzbox anzupingen - geht nicht! Diesen Bug gibt es bestimmt schon seit 10 Jahren. Es bekommt immer nur 1er (zufällig) Antworten. AVM's "Lösung" war dann der "stealth modus", durch den gar keine pings mehr beantwortet wurden ^^
